prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404 Practical
+Kivy GUI program for dynamic labels.
+
+"""
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+class DynamicLabels(App):
+    """Kivy App for showcasing dynamic labels."""
+
+    def __init__(self, **kwargs):
+        """Initialises dynamic labels instance with the keyword argument names."""
```

Comment:
Very minor detail, your docstrings (also in build function) aren't in imperative mood? (first word of the docstring)